integration/oidc/authorization-code/README.md

```diff
@@ -0,0 +1,197 @@
+# OpenID Connect Example with Server Side Web App
+## Overview
+This example application uses the OpenID Connect (OIDC) protocol to authenticate
+with Azure Active Directory and get an access token to request WorkflowGen's
+GraphQL API.
+
+## Prerequisites
+1. dotnet core sdk 2.2
+1. Make sure to have a valid WorkflowGen licence and serial number.
+1. Make sure to have access to an Azure subscription.
+1. Make sure to have a WorkflowGen server setup with OIDC authentication with
+Azure AD.
```

Comment:
Make sure to have a WorkflowGen server set up with OIDC authentication with Azure AD.